### PR TITLE
Intuit Mint sign up page

### PR DESCRIPTION
**Pull request by misterpaul4**
**Intuit Mint sign up page**

- we made it as responsive as we could

- we used proper form types and validated our html

- we added the footer elements to look exactly as it is on the project page

### CODE REVIEW

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning: 

Hey @misterpaul4 & @thedekerone 👋 

Good job so far! :tada: We have some few changes to do before we proceed (see review comment). 

***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.*** 👀

- Page Structure:
> Sign In Button
1. This project is required to have a [Sign In Button](https://accounts.intuit.com/signup.html) at the top right corner of the page.
> Create account Button:
2. The create account button is not styled properly. Kindly see to it that it resembles the original button.

- Section Specific:
1. Add the live link for your repo to the README file.
2. In the README, add a brief description on how to use your code, for anyone visiting the repo, thanks.
3. Remove all your inline styling in your markup and move them all to the CSS file. 
- When you have made the specified changes, please submit another code review. Thank you! 

*Happy coding 😄,*
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
...
+- Github: [@githubhandle](https://github.com/misterpaul4)
+- Twitter: [@twitterhandle](https://twitter.com/paulajuze)
+- Linkedin: [linkedin](https://www.linkedin.com/in/chukwuebuka-paul-ajuizeogu/)
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check the [issues page](issues/).
+
```
Comment (christianotieno) [inline README.md:42]: Add a live link to your repo and instructions on how to use your repo.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+.flex-between {
+  display: flex;
+  justify-content: space-between;
+}
+
+.flex-center {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```
Comment (ChristianOtieno) [inline style.css]: Good job with you CSS it is nice and clean.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Signup</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+  </head>
+  <body>
+    <main>
+      <img
+        width="140"
+        src="img/logo-intuit.png"
```
Comment (christianotieno) [inline index.html]: Remove all the inline styling in your markup and move them to the CSS file.

**Review by christianotieno — CHANGES_REQUESTED** [review]

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 🐱

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  TEAM 👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**

- [ ] OPTIONAL - If you want to improve your project a bit more, change the color of the links at the footer section to reflect that of the original page and also reduce the font-size of text in the footer section.

**THIS IS YOURS [Screenshot]👇**
ice_screenshot_20200326-192410


**THIS IS WHAT YOU HAVE ON THE ORIGINAL WEBPAGE👇[Screenshot]**
ice_screenshot_20200326-192424


Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**